build.gradle.kts

```diff
@@ -283,7 +283,7 @@ tasks.register("buildNdkIntegrationTestsArtifacts") {
     dependsOn(":instrumented:integration:assembleDebug")
 }
 
-nightlyTestsCoverageConfig(threshold = 0.86f)
+nightlyTestsCoverageConfig(threshold = 0.85f)
```

Comment:
we need instead update method signature in `WebViewTrackingE2ETests`, then we can keep 0.86 coverage.